src/app.js

```diff
@@ -264,6 +264,8 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
   }
 
   let appStore = new EntityStore('module', 'name')
+  // Get workspace before creating the App Manager
+  const workspace = JSON.parse(localStorage.getItem('workspace'))
```

Comment:
I feel that will throw if the key is not already there...

Reply:
It won't. I checked ;)